enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -113,6 +113,16 @@ The hook modifies the cgroup `kubepods.slice/kubepods-pod<pod-id>/crio-<containe
 It also modifies the cgroup`kubepods.slice/kubepods-pod<pod-id>/cpu.cfs_quota_us` value
 using the same formula.
 
+For cgroup v2 the same changes apply but with the respective to the API changes in v2, For example,
```

Comment:
Did we test that a PoC works with cgroups v2?

Reply:
@mrunalp I did and it works.
But it's important to realize that NTO, which is the component that responsible for deploying the plugin, doesn't support cgroupv2 just yet.
The plugin itself compatible with cgroupsv2 and it was tested independently of NTO.

Reply:
@Tal-or Can you provide more details on which part didn't work with NTO and cgroups v2?